types/tools/inline-tool.d.ts

```diff
@@ -43,6 +43,7 @@ export interface InlineTool extends BaseTool {
  */
 export interface InlineToolConstructorOptions {
   api: API;
+  data?: undefined;
```

Comment:
Inline Tools never accept `data` at the constructor. So I think it is not a solution.

Reply:
I also think it is good solution, but it works on current design without several changes.
I can't imagine better solution on current design.

Reply:
Me, too. 
This is the easiest way to fix the type definition.

Reply:
Or... We have plan to API changes for fixing about this case?